hover/effects/_keyframes.sass

```diff
@@ -0,0 +1,10 @@
+// Keyframes mixin to add browser prefixes to animations
+@mixin keyframes($animation-name)
+    @-webkit-keyframes #{$animation-name}
+        @content
+
+    @-moz-keyframes #{$animation-name}
```

Comment:
I don't think you need this. Firefox has been long supporting keyframes without the need to the vendor prefix